src/openssl/ssl/server.cr

```diff
@@ -72,6 +72,11 @@ class OpenSSL::SSL::Server
     end
   end
 
+  # This method allows to fetch the SSL SNI string if present
```

Comment:
`if present`? Does that mean this value is sometimes not available? What gets returned in that case? An empty string? Maybe that should be `nil`.
Also I think this method should be simply called `sni` because as far as I know in Crystal's method world you don't put `set` or `get` in front of method names.

Reply:
@r00ster91 done!